docs/how-to/add-custom-labels.md

```diff
@@ -0,0 +1,11 @@
+# How to add custom labels
+
+This charm supports adding custom labels to the runners.
+
+By using [`juju config`](https://juju.is/docs/juju/juju-config) to change the
+[charm configuration labels](https://charmhub.io/github-runner/configure#labels), additional 
+custom labels can be attached to the self-hosted runners.
+
+```shell
+juju config <APP_NAME> labels=<LABELS_CSV>
```

Comment:
Since CSV is a common file format, someone might think `LABELS_CSV` is a filename.
Would it be good to rename it to `LABELS` or `LABEL_LIST` and provide an example value?

Reply:
Updated, thank you!